packages/wonder-blocks-dropdown/components/single-select.js

```diff
@@ -111,6 +121,12 @@ export default class SingleSelect extends React.Component<Props, State> {
             open,
             keyboard,
         });
+
+        if (open) {
```

Comment:
This is a little confusing (my mind things, if we're open, we're about to close). Can you add a comment explaining that we just opened, so we need to tell folks we did (and vice versa for the close?